### PR TITLE
Use REACT components

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description
- Use React Components
- Ran linters check
- Update README

# Screenshots
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@
 > Describe between 1-3 key features of the application.
 
 - **[Display Home Page]**
+- **[Add UI design]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (zieeco) [inline README.md:54]: - [x] Please, let's update our readme to be professional by doing the following below:

- Let's remove all the placeholder text as these are meant to guide you on how to update the readme and are not expected to be in the readme after you've made the update to the readme.



Screenshot from 2023-02-14 14-17-20
Screenshot from 2023-02-14 14-17-03
Screenshot from 2023-02-14 14-16-55




Screenshot from 2023-02-14 14-20-00
Screenshot from 2023-02-14 14-19-53
Screenshot from 2023-02-14 14-19-47
Screenshot from 2023-02-14 14-19-32
Screenshot from 2023-02-14 14-18-47
Screenshot from 2023-02-14 14-18-01

--- 

- Please, let's remove this part also as we've not deployed the app

Screenshot from 2023-02-14 14-18-36

Screenshot from 2023-02-14 14-19-18

--- 

- Please, let's update the client to react and remove the rest.

Screenshot from 2023-02-14 14-17-49

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @joshualyngdoh ,


Great job so far 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (joshualyngdoh) [reply README.md:54]: Thank you for the feedback

**Review by DammyShittu — APPROVED** [review]
Hi @joshualyngdoh,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**